examples/workflow/nextjs/app/page.tsx

```diff
@@ -84,3 +84,11 @@ export default function Home() {
     </div>
   );
 }
+
+export default function HomePage() {
+  return (
+    <Suspense>
```

Comment:
What's up with Suspense without fallback 🤔

Reply:
was getting an error when I built the nextjs app https://nextjs.org/docs/messages/missing-suspense-with-csr-bailout